### PR TITLE
Build: Add Alpine docker to matrix

### PR DESCRIPTION
**Pull request by nschonni**
Related to #1589 
This "works" except that it Segfaults running `npm test` https://travis-ci.org/nschonni/node-sass/jobs/144365881#L3257

### CODE REVIEW

**Comment by saper** [comment]
Can you get a backtrace of this crash?

**Comment by nschonni** [comment]
@saper I'll see, but I'm not super familiar with Docker, and it seems to crash out the whole container when it dies. Trying to repoduce the issue locally isn't too consistent either.

**Comment by saper** [comment]
Hitting some resource limit? The next test in queue after `hey2` is [huge](https://github.com/sass/sass-spec/blob/master/spec/scss/huge/input.scss).

**Comment by nschonni** [comment]
@saper yeah I noticed that. I'm trying to get a repeatable crash locally with the docker beta and gdb.

File: .travis.yml [hunk 1]
```diff
@@ -47,6 +52,9 @@ before_install:
 script:
   - npm install
   - npm test
+  - if [ $TRAVIS_OS_NAME == "linux" ]; then
+      docker run --rm -i --env SKIP_SASS_BINARY_DOWNLOAD_FOR_CI=true -v $DOCKER_COPY:/opt/node-sass -w /opt/node-sass mhart/alpine-node:$TRAVIS_NODE_VERSION /bin/sh -c "apk add --no-cache python=2.7.12-r0 make gcc g++ && npm install --unsafe-perm --verbose && npm test";
```
Comment (xzyfer) [inline .travis.yml]: I would remove the `--verbose` flag. It crashes the travis ci interface.

Comment (nschonni) [inline .travis.yml]: Done. Must have left it while debugging. I'll probably move it over to a Dockerfile if I get this running properly

**Comment by xzyfer** [comment]
You'll also need to opt out for iojs builds.

File: .travis.yml [hunk 2]
```diff
@@ -37,6 +39,9 @@ before_install:
       export CXX="g++-4.7";
       export LINK="gcc-4.7";
       export LINKXX="g++-4.7";
+      export DOCKER_COPY="/tmp/alpine";
+      cp -r $TRAVIS_BUILD_DIR $DOCKER_COPY;
+      docker pull mhart/alpine-node:$TRAVIS_NODE_VERSION;
```
Comment (shouze) [inline .travis.yml]: There's not an official alpine image 🎉 
https://hub.docker.com/r/library/node/tags/6-alpine/

**Comment by serkanyersen** [comment]
any update on this?

**Comment by nschonni** [comment]
I'm going to close this one and take a look at getting actual docker files for those other OS we build for during the release.

**Pull request closed without merging**